### DISCUSSION TITLE
Dubbi e domande su impianto elettrico!

### DISCUSSION THREAD

**Discussion by esposimo** [indomus/forum#916] [Hardware & Firmware]
Ciao a tutti
ho scoperto il vostro sito da poco e mi ha allargato di molto gli orizzonti su ciò che riguarda la domotica, i componenti, ed home assistant in particolare (che già conoscevo). Complimenti!

Vengo al dunque, anche se non sarò breve
Sto per acquistare casa, e nei vari lavori di ristrutturazione ovviamente è incluso l'impianto elettrico, ed una LAN per cablare le varie camere.
La mia idea è quella di mettere su un HUB centrale con home assistant (su un mini pc con ubuntu) installandolo con docker, sul quale poi gireranno altri miei eventuali progetti.
Lavoro nell'IT quindi su questo aspetto non ho problemi, ma ho serie difficoltà su ciò che riguarda la parte elettrica.

Vengo quindi alle domande
Qualcuno ha mai provato a fare la domotica usando due reti in casa ? Cerco di essere chiaro
Vorrei avere due LAN in casa
Una esposta dal modem/router del provider di connessione alla quale saranno connessi i dispositivi che si utilizzano tutti i giorni (pc/tablet/tv/console/etc).
Una LAN gestita da un altro router (comunque connesso al modem principale per uscire su internet), che mi permetta, attraverso una diversa classe di ip, di isolare tutti i sensori (che supportano la lan) che andrò ad installare

Quindi in parole povere
una 192.168.1.0/24 sul router/modem
una 10.10.10.0/24? sul router in cascata, che esce su internet tramite la 192.168.1.0
Ovviamente l'hub, sarà connesso tramite cavo (con impianto di rete che farò fare) ad entrambe le reti.
Così sarà gestibile tramite la rete di casa (da smartphone, pc, tablet) ma comunicherà con i sensori su rete separata.

Ora le domande che riguardano la parte elettronica
La mia idea è di "predisporre" la casa con home assistant e sensori di energia, per poi allargare le funzionalità una volta che vado ad abitarci (comprando sensori esterni, telecamere, etc, in base a ciò che voglio fare insomma)

- Ho puntato i dispositivi shelly per le prese elettriche, che mi consentono di effettuare il monitoraggio dei consumi.
Vorrei avere informazioni sui consumi di ogni singola presa, e non a "gruppo" o scatola/503.
Quindi, se ad es. in soggiorno faccio mettere 4 prese (tra tv e console le finisco subito), come gestisco il controllo di ognuna, dato che lo shelly occupa lo spazio di un frutto e riesce a monitorare soltanto una presa ?
Conviene, monitorare ogni singola spina? O vado a gruppi e quindi monitoro la "presa" soggiorno, la "presa" corridoio, etc etc.
Inoltre, nel caso in cui io inserisca una scarpetta elettrica (ovviamente su di essa non metterò gli elettrodomestici tutti insieme, ma apparecchiature tipo computer + monitor + un caricabatterie magari), lo shelly regge?

- In soggiorno vorrei mettere 4/6 faretti (è un ambiente unico soggiorno+cucina). Come gestisco in questo caso il tutto?
Se in una 503 metto uno shelly (per attivazione luci e consumi) ed un semplice pulsante per accensione, avrò bisogno di 4 scatole 503 per gestire ogni singola luce?
Inoltre, dato che le luci possono essere attivate sia con la domotica che con l'interruttore classico, non rischierei di ritrovarmi poi l'interruttore in uno stato non congruo rispetto allo stato delle luci?
Es. accendo con l'interruttore, spengo con l'assistente, ma il pulsante resta comunque su on, se lo vado a premere ovviamente va su off, e devo ripremere per portarlo su on.

Perdonate le domande sull'elettronica, ma sono molto scarso in questo ambito e devo capirci qualcosina per non trovarmi impreparato

Grazie anticipatamente!

**Comment by Silvano2161** [comment]
Ciao, anche io ho montato dei componenti Shelly per la gestione di alcune prese e luci, anche se non a livello punto-punto come vuoi fare tu....
E' vero che un modulo Shelly occupa lo spazio di un componente "elettrico" tipo interruttore o presa italiana, ma come scatole di montaggio oltre le 503 da tre moduli esistono le 504, 506, 507 su cui si possono montare più moduli, per cui basta scegliere la scatola delle dimensioni che ti serve per farci stare anche i moduli Shelly.

Al riguardo dei comandi per le luci (ma vale anche per le prese), l'uso di Shelly 2,5 ti risparmia uno spazio, con un solo modulo comandi due gruppi di luci in modo totalmente indipendente, inoltre per evitare il problema delle posizioni di "Acceso/spento" degli interruttori è sufficiente montare dei pulsanti che si usano normalmente in accoppiamento ai relè passo-passo (del tipo che con una pressione la luce si accende, con una successiva pressione del pulsante la luce si spegne) e configurare lo Shelly in modalità "pulsante" anzichè interruttore.
In questo modo risolvi anche il problema dei deviatori e il proliferare di collegamenti elettrici, bastano due fili a cui collegare in parallelo tutti i pulsanti (da 1 a n a piacere) ed il gioco è fatto.
Anche lo Shelly 2,5 è un grado di monitorare l'energia in transito, e lo fa in modo indipendente per ogni canale

La portata in corrente dei moduli Shelly è generalmente dichiarata da 16 Ampere, e può darsi che il contatto del relè contenuto all'interno possa farcela, quello che mi lascia perplesso piuttosto è la dimensione dei morsetti a cui collegare i fili, non si riesce in alcun modo a collegarci fili di sezione maggiore di 1,5 mm2, per cui carichi di oltre 1000 Watt (corrispondenti ad oltre 4 Ampere di corrente) eviterei di collegarli.
Tieni conto che di norma sarebbe bene usare fili da 2,5 mm2 per alimentare le prese di corrente, ma se non ci attacchi stufe o forni elettrici, o lavatrici ecc.. insomma roba che richiede almeno 2kW di potenza puoi usare anche i fili da 1,5 mm2.

In alternativa ci sono gli Shelly EM, che non sono interessati dal passaggio di corrente, ma usano delle sonde esterne che vanno inserite intorno al filo di fase, e hanno portate da 50 Ampere oppure 120 Ampere, che in casa non ci saranno mai neanche in caso di cortocircuito.
Questi li uso per monitorare il consumo della casa, e siccome ho un impianto fotovoltaico, tengo d'occhio anche la produzione e lo scambio.

Spero di esserti stato utile.

Silvano

**Comment by esposimo** [comment]
Ciao

si mi sei stato di aiuto, ma al momento ho una marea di informazioni da valutare e poche conoscenze in ambito elettrico per capire come implementare il tutto (mi farò aiutare da un elettricista ovviamente).
Con le tue informazioni potrei pensare ad una cosa del genere
Per le prese dove so che non metterò elettrodomestici, shelly 2.5 (una 503 con due prese ed uno shelly e la gestisco per intero. Magari in soggiorno dove metterò la postazione tv metterò una 506 da 6 frutti, 4 prese e due shelly 2.5

Per le prese dove vanno gli elettrodomestici (che pure quelli attacchi,e non stacchi più, frigo, forno, etc), Shelly EM, per stare tranquillo.

Per le luci, vorrei usare i Dimmer per regolare anche l'intensità (quelli con l'attacco Neutro, che hanno il misuratore di potenza).

Pensavo, inoltre, che magari si potrebbe fare una cassetta dedicata in un punto della casa dove convogliare tutti gli Shelly, in modo tale da farci "manutenzione" in maniera più semplice, anzichè smontare le placche delle 503 etc.

**Comment by NdK73** [comment]
Ciao.

Per mia esperienza, vedo diversi problemi.

Innanzi tutto, a meno di non usare shelly Pro (i nuovi, e solo alcuni), la connessione è WiFi e non Ethernet: rischi di far stendere cavi "inutilmente". E all'aumentare dei dispositivi servono AP sempre più costosi (molti AP economici vanno in crisi già con una ventina di client contemporanei! :( ). Da tenere presente per non avere troppi problemi dopo.

Altro punto: la portata dei contatti dei relè non è da intendere come valore "a regime": se attacchi un carico con cos-phi basso, bruci il relè in poche operazioni anche se la corrente "a regime" è abbondantemente più bassa. P.e. potresti avere una pompa da 1HP (750W): molto meno dei 16A, vero? In realtà, quando il motore (carico induttivo) deve partire, assorbe molto di più. Anche per questo, anche se i relè sono marcati 16A nei connettori non si riescono ad infilare fili da 2.5mm².

Ultimo ma non meno importante: la certificazione dell'impianto. Si, concordo che è una cavolata, ma meglio pensarci comunque. L'elettricista, ogni volta che fa una modifica all'impianto, te lo deve certificare. Se fai una modifica tu (tipo aggiungere una seconda presa dove ce n'era solo una), l'impianto perde la certificazione e in caso di problemi sei nei guai. Il "succo" è che cosa si intende per modifica: in soldoni, se i tuoi apparati domotici sono collegati ad una spina che si inserisce in una presa, sei a posto. Se gli stessi apparati li infili all'interno della scatola prima della presa, devi far ri-certificare l'impianto. Ridicolo, vero? Comunque, per questo ho in giro parecchie BW-SHP6 e simili: sono prese passanti che posso inserire dove servono, non occupano quasi nulla e non modificano l'impianto.

**Reply by esposimo** [reply]
Ciao, ti rispondo cercando di chiarirti le idee in merito a quanto richiesto

Per quanto riguarda la rete, la mia idea è questa
Rete di casa (192.168.1.1/24), con modem del provider (o mio) che viene utilizzato solo ed esclusivamente dai dispositivi di casa (computer ,smartphone, tablet, televisori, HUB con HA)
Rete "protetta" (es. 10.10.10.0/24), con un router, sul quale agganciare tutti i dispositivi smart di controllo (es. gli Shelly) in wifi. Questo router poi, è agganciato al modem/router principale per uscire su internet, laddove serve. Il computer che mi fa da HA, è agganciato ad entrambi le reti, con cavo.
Fondamentalmente per l'impianto di rete ho in mente di portare un cavo in salone, uno in camera da letto, uno nello studio. Nel salone ci metterò televisore e le console da gioco, posso usare anche uno switch con la singola uscita di rete che mi porto, non è un problema, tanto non uso mai + console contemporaneamente, e se guardo qualcosa in streaming sulla tv comunque non sto usando banda con le console. Avrei comunque la fibra FTTH, quindi non vedo problemi di banda Nell'avere più dispositivi dietro uno switch.
In camera da letto solo per la smart TV.
Un cavo in fondo al corridoio, per un AP.
Nello studio mi servono 4 cavi di rete (uno per angolo, se mi vien voglia di cambiare disposizione dato che lavoro in smart fisso) ma userò sempre e solo una porta ethernet. In questo caso potrei anche portare un solo cavo di rete dall'ingresso allo studio, e montare uno switch nascosto con i cavi che vanno in ogni angolo (sempre nascosti) ma non mi troverò a passare montagne di fili dall'ingresso (dove ho il router) allo studio.
Altra nota, vorrei portare anche un cavo dal 2° router dall'ingresso allo studio per l'HUB. Fondamentalmente l'HUB avrà due ethernet, per attaccarsi ad entrambi i router e stare su entrambi le reti. Così è visibile sia dalla rete dei componenti smart, che da quella di "casa".  Questo perchè mi ritroverei da un lato la rete con i dispositivi smart di monitoraggio, sull'altra rete altre componenti ugualmente smart (tv, console)

Per quanto riguarda il monitoraggio dei consumi:
Si sono consapevole del problema, ma la mia idea è quella di usare gli Shelly 1PM/Shelly Dimmer per quei punti dove so che non ci sono carichi alti (punti luce con regolazione potenza, prese per televisori, etc) e per quei posti dove so che vanno elettrodomestici, caldaia, e altra roba che consuma molto, usare lo Shelly EM che arriva a 50A.
Fondamentalmente a me serve solo il monitoraggio di consumi, già la caratteristica di staccare la presa la ritengo "inutile" in quanto per quei dispositivi come forno, lavatrice, ma anche la TV, il computer, non c'è mai lo spegnimento "staccando" la spina, ma un semplice power off. Quindi è il dispositivo che deve essere "smart". Se disattivo uno shelly dove c'è la TV e poi lo attivo, la TV non si accende, si accende solo il LED. Idem un computer.
Non ho trovato nulla che funga solo da "monitoraggio" dei consumi per una singola presa e che sia smart.
La presa che riporti tu realizza il mio scopo ma mi ritroverei ad averne tante ed esteticamente non mi gusta.

Quindi, riepilogando, la mia idea è:
Dispositivi di monitoraggio consumi su tutte le prese e punti luce (adatti ovviamente a sostenere il carico in base allo scopo della presa)
Doppia rete, una per i dispositivi di monitoraggio, una per il resto.
HUB agganciato ad entrambi le reti (con cavo) 

Il tutto ovviamente avallato da un elettricista che ne capisce, con il quale al momento ancora devo parlare. Magari mi dice che sto facendo una cosa inutile.

**Reply by NdK73** [reply]
Forse sono stato troppo criptico: gli shelly (non pro) li connetti solo in WiFi a 2.4G. Quindi, più che portare la rete cablata, il problema è la copertura wireless. Problema su vari fronti:
1) come detto, la capacità di gestire tanti client; non è questione di banda (ridicola, per la domotica) ma di memoria a bordo dell'AP (ipersemplificando, ogni client richiede un tot di RAM perché l'AP ne riesca a mantenere lo stato)
2) interferenza con altri AP nelle vicinanze (sulla banda dei 2.4 puoi usare senza interferenze solo i canali 1, 6 e 12)

Se per la rete porti cavi cat 7 sei decisamente a posto. Se poi usi switch managed (che però costano più degli altri) puoi definire diverse VLAN per segregare il traffico quasi come se fosse su cavi diversi (a parte la banda, ma come detto questa non è un problema).
Come prese ce ne sono anche di più carine e talmente ridotte da essere quasi invisibili, tipo le ZEP2-10A (ne ho avute di altra marca).

L'utilità di avere il monitoraggio globale e un po' di controlli sugli apparecchi più energivori è che puoi automatizzare lo stacco prioritizzando i carichi: se in totale stai già consumando 2kW e viene accesa la lavatrice, spegni immediatamente la sua presa così non ti salta il contatore. E chi l'ha accesa si rende immediatamente conto che erano già accesi il forno e la lavastoviglie...

Per quel che riguarda la doppia rete, con me sfondi una porta aperta... Io ne ho 3: quella a valle del router del provider (insicura) dove apossono accedere gli ospiti generici, una per gli ospiti più affidabili e i device in cloud, e infine una per casa, dove ho anche tutti i device gestiti da HA. :) Tutto VLAN-based.

**Reply by esposimo** [reply]
La mia idea era infatti di usare 2 punti di accesso per gli Shelly.
Sul router cosiddetto "privato" o sull'AP.

Questo perchè, sia per le dimensioni/disposizione della casa, sia per il problema da te elencato, su un totale di..20 shelly ? Dividerei il carico su 2 AP, 10 per uno.

Poi se esistono altri metodi per monitorare il consumo di ogni apparecchio, ben vengano i suggerimenti

**Reply by NdK73** [reply]
Se hai bisogno di monitoraggio più centralizzato, cerca SDM-120C (ce ne sono anche altri, comunque). Però esce in RS485, ma con un "gateway" (puoi farlo con un ESP8266 senza particolari problemi) puoi monitorare fino a 126 linee :)

**Reply by antony84** [reply]
Ciao a tutti, leggevo questa interessante discussione perché anche io sto cercando di capire come realizzare un nuovo impianto elettrico con gli shelly, e come cablare casa con eth e wifi. Si tratta di una casa singola si due piani.
Avevo puntato subito i dispositivi PRO per poi però rendermi conto essenzialmente di due cose: 1- costano un 70% in più dei plus wifi; 2- bagno portati i fili di tutti gli interruttori fino al QG, dove ci sono appunto i PRO.

Mi chiedo se ne valga davvero la pena spendere di più per i pro o è meglio usare i wifi plus e spendere qualcosa in più per la gestione del wifi interno, tipo due access point per piano invece che uno.

Tra l'altro se non sbaglio per le luci dimmerabili (strisce led e faretti) non ci sono dispositivi PRO da quadro e lan, quindi per queste andrebbe messo un dimmer 2 wifi giusto?

**Reply by KC-inDomus** [reply]
personalmente uso gli standard integrati con Home Assistant e non sento il benché minimo bisogno di passare ai PRO. oggi ricomprerei gli standard

**Comment by NdK73** [comment]
Se si hanno veramente tanti dispositivi WiFi nella rete (più di 60,
indicativamente... e si fa abbastanza in fretta ad arrivarci), il router
potrebbe arrivare ad avere problemi. Anche con sistemi mesh. Per questi
casi, probabilmente è da valutare il passaggio all'uso di zigbee (o
simili), che oltretutto potrebbero risolvere alcuni problemi di scarsa
copertura.

In ogni caso, consiglio di impostare le regole "base" sugli shelly (o nei
nodi zb), così che p.e. si possa accendere la luce anche se HA è in
manutenzione. E magari può valere la pena di avere anche un AP di riserva,
magari sdozzo e minimale, caso mai si guasti quello principale. Qualcosa
che basti accendere, senza dover fare altro.

Diego

Il mer 14 dic 2022, 08:44 KC-inDomus ***@***.***> ha scritto:

**Reply by KC-inDomus** [reply]
> Se si hanno veramente tanti dispositivi WiFi nella rete (più di 60, indicativamente... e si fa abbastanza in fretta ad arrivarci), il router potrebbe arrivare ad avere problemi. Anche con sistemi mesh. Per questi casi, probabilmente è da valutare il passaggio all'uso di zigbee (o simili), che oltretutto potrebbero risolvere alcuni problemi di scarsa copertura. 

Io ne ho oltre 50-60 sul Wi-Fi, e non ho mai mezzo problema. ZigBee va benissimo per i sensori, per gli attuatori sinceramente mai e poi mai li adotterei su standard diversi da Wi-Fi, o per lo meno nessuno che preveda un bridge in mezzo.

**Reply by antony84** [reply]
Al di là se gli AP ce la fanno a tenere tutti questi dispositivi, ma tutte ste onde radio sparse per casa è una cosa positiva per la salute?

**Reply by KC-inDomus** [reply]
mah, non saprei. personalmente trovo particolarmente utile il terzo occhio che mi è cresciuto in fronte, però boh

**Comment by NdK73** [comment]
In generale concordo nell'essere refrattario ai bridge, ma usando un bridge
open (ZigStar Lan gateway) e backuppando spesso la config zb (almeno dopo
ogni aggiunta di device) mi tranquillizzo un tantino e lo ritengo
accettabile.
Sarebbe carino che ZHA supportasse più gw, per quelle situazioni dove è
effettivamente utile, senza bisogno di istanze multiple di HA.

Diego

Il mer 14 dic 2022, 10:42 KC-inDomus ***@***.***> ha scritto:

**Reply by snakuzzo** [reply]
posso chiederti cortesemente di evitare di fare reply direttamente dalla mail, evitando così di creare ogni volta nuovi thread?
Grazie.

**Reply by NdK73** [reply]
Sorry, non avevo notato. E onestamente pensavo che GH avesse una gestione del threading via mail più evoluta...
